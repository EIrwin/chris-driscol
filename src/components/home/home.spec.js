// @flow
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Component from './home-page.component';

const fixtures = {
  chris: {
    social: {},
    about: {},
    skills: {},
    experience: [],
    work: [],
  },
};

it('renders correctly', () => {
  const renderer = new ShallowRenderer();
  const tree = renderer.render(<Component {...fixtures} />);
  expect(tree).toMatchSnapshot();
});
