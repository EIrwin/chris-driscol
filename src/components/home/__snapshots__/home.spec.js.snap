// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <Navigation
    social={Object {}}
  />
  <Header />
  <About
    about={Object {}}
  />
  <Skills
    skills={Object {}}
  />
  <Experience
    experiences={Array []}
  />
  <Portfolio
    works={Array []}
  />
  <Contact
    viewer={
      Object {
        "about": Object {},
        "experience": Array [],
        "skills": Object {},
        "social": Object {},
        "work": Array [],
      }
    }
  />
  <Footer
    social={Object {}}
  />
</div>
`;
